extensions/jackson/deployment/src/main/java/io/quarkus/jackson/deployment/IgnoreJsonDeserializeClassBuildItem.java

```diff
@@ -10,13 +12,22 @@
  */
 public final class IgnoreJsonDeserializeClassBuildItem extends MultiBuildItem {
 
-    private final DotName dotName;
+    private final List<DotName> dotNames;
 
     public IgnoreJsonDeserializeClassBuildItem(DotName dotName) {
-        this.dotName = dotName;
+        this.dotNames = List.of(dotName);
+    }
+
+    public IgnoreJsonDeserializeClassBuildItem(List<DotName> dotNames) {
+        this.dotNames = dotNames;
     }
 
+    @Deprecated(forRemoval = true)
     public DotName getDotName() {
-        return dotName;
+        return dotNames.get(0);
```

Comment:
Just occurred to me that this may throw an exception if the list is empty

Reply:
It shouldn't happen, but yeah, let's guard against it